landscaper/pkg/controlplane/apis/imports/v1alpha1/zz_generated.defaults.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
This should not be here. Please update your local Golang version and run `make generate` again

Reply:
I ran it